doc/specs/application-auth-token.md

```diff
@@ -48,17 +129,47 @@ The proper way to sign the token is as follows:
 3. Sign with ed25519 ECDSA
 4. HexEncode the result bytes into a string
 
-```text
-AAT {
+```bash
+  AAT = {
     ApplicationSignature: "",
     ApplicationPublicKey: a.ApplicationPublicKey,
     ClientPublicKey:      a.ClientPublicKey,
     Version:              a.Version,
-}
+  }
+  AATBytes = JSON.Encode(AAT)
+  Message = SHA3_256(AATBytes)
+  AAT.ApplicationSignature = ED25519.Sign(Message)
 ```
 
-`AATBytes = JSON.Encode(AAT)`
+### Relay Generation & Signing using AAT
 
-`Message = SHA3-256(AATBytes)`
+This section can be summarized by two key points:
 
-`AAT.Signature = ED25519.Sign(Message)`
+- AAT includes client public key, signed by application private key
+- Relay request includes an AAT, signed by client private key
+
+The `Client` is needed to sign the relays while the `Application` is the one that
```

Comment:
How about rephrasing the first sentence like this?  It would be good to mention the max relays.

The `Client`, an application to send relays, needs to sign every relay request, while `Application` needs to be staked once with tokens on-chain to get access to Pocket Network's services.  In every session, the maximum number of relays for an app is proportional to the tokens to be staked with the app.  To make more relays, you need to stake more tokens.